src/main/java/org/broadinstitute/consent/http/health/SendGridStatus.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.consent.http.health;
+
+import com.codahale.metrics.health.HealthCheck.Result;
+import com.google.gson.Gson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SendGridStatus {
+    private Object page;
+    private Object status;
+
+    public Object getPage() {
+        return page;
+    }
+
+    public void setPage(Object page) {
+        this.page = page;
+    }
+
+    public Object getStatus() {
+        return status;
+    }
+
+    public void setStatus(String indicator, String description) {
+        Map<String, String> statusMap = new HashMap<>();
+        statusMap.put("indicator", indicator);
+        statusMap.put("description", description);
+        status = new Gson().toJson(statusMap);
+    }
+
+    private Map<String, String> getStatusMap() {
+        return new Gson().fromJson(getStatus().toString(), Map.class);
+    }
```

Comment:
This is an example of something that should stick out in your mind when writing code. This map is, in essence, a sort of temporary variable. Temp variables are very often replaceable by proper object modeling. In this case, if the `status` field were a proper class with an indicator field, you wouldn't need this method at all.